cmd/ltctl/main.go

```diff
@@ -47,6 +47,19 @@ func RunDestroyCmdF(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("failed to create terraform engine: %w", err)
 	}
+
+	// If we are using a Terraform managed DB.
+	if config.ExternalDBSettings.DataSource == "" {
```

Comment:
We could also use `t.output.HasDB()`, which seems more idiomatic and we'd avoid the comment, but then you'd need to run `t.loadOutput()` before that (or do it inside `t.Destroy()`, which already loads the output)

Reply:
Yeah, I think it's okay.